engine/core/pom.xml

```diff
@@ -232,6 +226,11 @@
       <artifactId>contiperf</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.javatlacati</groupId>
+      <artifactId>contiperf</artifactId>
```

Comment:
What is this and why is it needed? Should this have a scope of test?

Reply:
These were eralier refered from "org.databene.contiperf" but with the Unit leatest version updates, it was throwing issues like "java.lang.RuntimeException: java.lang.NoSuchFieldException: fNext" and had to chege the dependency to this with the updated fix compatible with newer junit version.